### ISSUE TITLE
PyCharm IDE error when trying to format a table while cursor is not within table

### ISSUE THREAD

**Issue by denis-roy** [tadashi-aikawa/markowl#5]
Version affected: 0.7.2

Steps to reproduce:
- Create a Markdown file
- Create a Markdown table
- Populate some rows
- Move cursor outside of table
- Hit <kbd>ALT</kbd>+<kbd>;</kbd> to format table

Result:
PyCharm will generate an IDE error with the following `stacktrace.txt`

```
java.lang.IllegalArgumentException: Invalid range specified: (1677, 1675); 
	at com.intellij.openapi.util.TextRange.assertProperRange(TextRange.java:222)
	at com.intellij.openapi.util.TextRange.assertProperRange(TextRange.java:217)
	at com.intellij.openapi.util.TextRange.assertProperRange(TextRange.java:213)
	at com.intellij.openapi.util.TextRange.<init>(TextRange.java:41)
	at com.intellij.openapi.util.TextRange.<init>(TextRange.java:30)
	at net.mamansoft.markowl.action.markdown.FormatTableActionKt.getRangeAsTable(FormatTableAction.kt:36)
	at net.mamansoft.markowl.action.markdown.FormatTableAction$actionPerformed$1.invoke(FormatTableAction.kt:48)
	at net.mamansoft.markowl.action.markdown.FormatTableAction$actionPerformed$1.invoke(FormatTableAction.kt:39)
	at net.mamansoft.markowl.domain.OwlDocument$sam$java_lang_Runnable$0.run(OwlDocument.kt)
	at com.intellij.openapi.command.WriteCommandAction.lambda$runWriteCommandAction$4(WriteCommandAction.java:348)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$1(WriteCommandAction.java:143)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:936)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$2(WriteCommandAction.java:141)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:216)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:182)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.doRunWriteCommandAction(WriteCommandAction.java:150)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.run(WriteCommandAction.java:117)
	at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:348)
	at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:336)
	at net.mamansoft.markowl.domain.OwlDocument.action(OwlDocument.kt:60)
	at net.mamansoft.markowl.action.markdown.FormatTableAction.actionPerformed(FormatTableAction.kt:47)
	at com.intellij.openapi.keymap.impl.ActionProcessor.performAction(ActionProcessor.java:65)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$1.performAction(IdeKeyEventDispatcher.java:563)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.lambda$doPerformActionInner$8(IdeKeyEventDispatcher.java:677)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:261)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.doPerformActionInner(IdeKeyEventDispatcher.java:670)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:603)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:574)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processActionOrWaitSecondStroke(IdeKeyEventDispatcher.java:457)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.inInitState(IdeKeyEventDispatcher.java:450)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:218)
	at com.intellij.ide.IdeEventQueue.dispatchKeyEvent(IdeEventQueue.java:800)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:750)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:443)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:825)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:442)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:794)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:494)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

Thank you!

**Comment by tadashi-aikawa** [comment]
@denis-roy 

Thank you for a bug report! I will fix it 👍

**Comment by tadashi-aikawa** [comment]
I have released a new version v0.7.3.

It would probably be tomorrow or day after tomorrow that you can upgrade it because JetBrains needs to approve it.

Best regards.

**Comment by denis-roy** [comment]
No worries at all, thank you for the prompt fix M. Aikawa
